torchci/rockset/metrics/__sql/master_commit_red_avg.sql

```diff
@@ -12,8 +12,7 @@ WITH all_jobs AS (
         AND job.name != 'generate-test-matrix'
         AND ( -- Limit it to workflows which block viable/strict upgrades
             ARRAY_CONTAINS(SPLIT(:workflowNames, ','), LOWER(workflow.name))
-            OR workflow.name like 'linux-binary%'
-            OR workflow.name like 'windows-binary%'
+            OR workflow.name like 'linux-binary%'            
```

Comment:
updating this query will temporarily change the hud stats, but the stats will revert to including the "skipped" windows-binary failures once this change is reverted

Reply:
Update: https://github.com/pytorch/pytorch/pull/100638 has been given the green light to not run Windows binary jobs on every PR as they are redundant and have hight correlation with the rest of Windows CI jobs already run.